pkgs/development/python-modules/transformers/default.nix

```diff
@@ -0,0 +1,36 @@
+{ buildPythonPackage
+, stdenv
+, fetchFromGitHub
+, sacremoses
+, requests
+, sentencepiece
+, boto3
+, tqdm
+, regex
+, numpy
+}:
+
+buildPythonPackage rec {
+  pname = "transformers";
+  version = "2.2.1";
+
+  src = fetchFromGitHub {
+    owner = "huggingface";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1p8p3lhhiyk1xl9gpgq4vbchyz57v3w7hhvsj1r90zs3cckindl8";
+  };
+
+  # Tests disabled because requires downloading additional files from s3
+  # as well as having tensorflow and/or pytorch installed to perform them.
+  doCheck = false;
```

Comment:
please enable

```
  checkInputs = [ pytest ];
  # pretrained tries to download from s3
  checkPhase = ''
    cd transformers # avoid importing local files
    HOME=$TMPDIR pytest -k 'not pretrained_tokenizers'
  '';
```

Reply:
yup, done - thank you. 

I seem to be missing a trick here, how did you quickly find the substring for which tests to exclude? For both transformers and sacremoses? Thanks!

Reply:
I also had a few more questions about general nix development patterns, like how to best keep your system on one release and have another repo for development, but be able to use these derivations straight after commiting? 

And a few others... is there a recommended place to ask them or we can talk somewhere? Thank you!

Reply:
the -k arguments will just filter based on a substring of any test. I just ran the entire test suite, saw which failed, and determined whether it was an issue due to the sandbox environment or an issue with the package. In this case, that one test failure just because of the sandbox environment which is fine.